MapboxDirections/MBDirections.swift

```diff
@@ -48,70 +48,70 @@ let userAgent: String = {
-        
+
         If the request was canceled or there was an error obtaining the routes, this parameter may be `nil`.
      - parameter routes: An array of `Route` objects. The preferred route is first; any alternative routes come next if the `RouteOptions` object’s `includesAlternativeRoutes` property was set to `true`. The preferred route depends on the route options object’s `profileIdentifier` property.
-        
+
         If the request was canceled or there was an error obtaining the routes, this parameter is `nil`. This is not to be confused with the situation in which no results were found, in which case the array is present but empty.
      - parameter error: The error that occurred, or `nil` if the placemarks were obtained successfully.
      */
     public typealias RouteCompletionHandler = (_ waypoints: [Waypoint]?, _ routes: [Route]?, _ error: NSError?) -> Void
-    
+
     /**
      A closure (block) to be called when a map matching request is complete.
-     
+
      If the request was canceled or there was an error obtaining the matches, this parameter is `nil`. This is not to be confused with the situation in which no matches were found, in which case the array is present but empty.
      - parameter error: The error that occurred, or `nil` if the placemarks were obtained successfully.
      */
     public typealias MatchCompletionHandler = (_ matches: [Match]?, _ error: NSError?) -> Void
-    
+
     // MARK: Creating a Directions Object
-    
+
     /**
      The shared directions object.
-     
+
      To use this object, a Mapbox [access token](https://www.mapbox.com/help/define-access-token/) should be specified in the `MGLMapboxAccessToken` key in the main application bundle’s Info.plist.
      */
     @objc(sharedDirections)
     public static let shared = Directions(accessToken: nil)
-    
+
     /**
      The API endpoint to use when requesting directions.
      */
     @objc public private(set) var apiEndpoint: URL
-    
+
     /**
      The Mapbox access token to associate with the request.
      */
     @objc public let accessToken: String
-    
+
     /**
      Initializes a newly created directions object with an optional access token and host.
-     
+
      - parameter accessToken: A Mapbox [access token](https://www.mapbox.com/help/define-access-token/). If an access token is not specified when initializing the directions object, it should be specified in the `MGLMapboxAccessToken` key in the main application bundle’s Info.plist.
-     - parameter host: An optional hostname to the server API. The [Mapbox Directions API](https://www.mapbox.com/api-documentation/?language=Swift#directions) endpoint is used by default.
+     - parameter host: An optional hostname to the server API. The [Mapbox Directions API](https://www.mapbox.com/api-documentation/navigation/#directions) endpoint is used by default.
```

Comment:
Will there be any language-specific content in the new documentation page? The `language=Swift` parameter automatically switches any example code to Swift, since most people reading these documentation comments are coding in Swift. However, if the new page doesn’t include any example code, then this change is fine.

Reply:
No, there won't be any lang-specific content on the new pages. Instead, we're having the cURL example only and then linking out to the SDKs or libraries when appropriate. That way the SDK reference docs will be the Source of Truth™️ for examples 😁